core/templates/services/schema-default-value.service.spec.ts

```diff
@@ -140,10 +139,19 @@ describe('Schema Default Value Service', () => {
     var loggerErrorSpy = spyOn(ls, 'error').and.callThrough();
     const schema = {
       type: 'invalid'
-    };
+    } as unknown as Schema;
 
```

Comment:
@vojtechjelinek We'll have to use `as unkown as X` here because `type: invalid` does not exist on schema as a type and we can't pass it to `getDefaultValue` function

Reply:
Hmm, can be maybe use `@ts-expect-error` and add a comment explaining why?

Reply:
done